requirements/dev.in

```diff
@@ -0,0 +1,3 @@
+-r base.in
```

Comment:
This is not used anywhere so don't add it.

Reply:
Addressed 1e6218c61a3493c50c7a0708b0f6546f343bfe76

Reply:
Marking this as resolved.